### PR TITLE
SE-2252: add simple val

### PR DESCRIPTION
**Pull request by slemasne**

### CODE REVIEW

File: examples/lusid/run-valuation/5-run-simple-valuation.sql [hunk 1]
```diff
@@ -0,0 +1,31 @@
+@measure =
```
Comment (DavidWassellFinbourne) [inline examples/lusid/run-valuation/5-run-simple-valuation.sql:1]: There are no comments for the next 4 sections of sql?

File: examples/lusid/run-valuation/2-create-portfolio.sql [hunk 2]
```diff
@@ -0,0 +1,14 @@
+-- Defining scope and code variables
+@@portfolioScope = select 'ibor';
+@@portfolioCode1 = select 'EQUITY_UK';
```
Comment (fox8090) [inline examples/lusid/run-valuation/2-create-portfolio.sql:3]: Should this be 'IBOR' and 'uk-equity' to match the scope and code in create-transaction-portfolio.sql? Or are they intentionally different?

File: examples/lusid/run-valuation/2-create-portfolio.sql [hunk 3]
```diff
@@ -0,0 +1,14 @@
+-- Defining scope and code variables
+@@portfolioScope = select 'ibor';
+@@portfolioCode1 = select 'EQUITY_UK';
+
+-- Define the portfolio data
+@create_portfolio = select 'Transaction' as PortfolioType, @@portfolioScope as PortfolioScope, @@portfolioCode1 as
+   PortfolioCode, @@portfolioCode1 as DisplayName, '' as Description, #2000-01-01# as Created, ''
+   as SubHoldingKeys, 'GBP' as BaseCurrency;
```
Comment (fox8090) [inline examples/lusid/run-valuation/2-create-portfolio.sql:8]: Very minor thing, would it be better to have each value on a new line to match the formatting of the other files in the folder?

File: examples/lusid/run-valuation/README.md [hunk 4]
```diff
@@ -0,0 +1,21 @@
+# Running valuations
```
Comment (DavidWassellFinbourne) [inline examples/lusid/run-valuation/README.md:1]: Minor comment: Does it read better to say "in drive" rather than "on drive" (maybe just me)?

**Review by DavidWassellFinbourne — COMMENTED** [review]
Just a few comments (all minor and layout related)

**Review by fox8090 — COMMENTED** [review]
Looks good! A couple of minor suggestions but apart from that I think its great.

File: examples/lusid/run-valuation/1-create-instruments.sql [hunk 5]
```diff
@@ -0,0 +1,18 @@
+-- Load worksheet from Excel file which conatins instrument data
```
[the remark this answers is not in the document]
Comment (slemasne) [reply examples/lusid/run-valuation/1-create-instruments.sql:1]: Yeah, agreed.

[the remark this answers is not in the document]
Comment (slemasne) [reply examples/lusid/run-valuation/1-create-instruments.sql:1]: Agreed we should keep it consistent.

File: examples/lusid/run-valuation/1-create-instruments.sql [hunk 6]
```diff
@@ -0,0 +1,18 @@
+-- Load worksheet from Excel file which conatins instrument data
+
```
[the remark this answers is not in the document]
Comment (slemasne) [reply examples/lusid/run-valuation/1-create-instruments.sql:2]: Thanks.

File: examples/lusid/run-valuation/1-create-instruments.sql [hunk 7]
```diff
@@ -0,0 +1,18 @@
...
+@instruments_data = use Drive.Excel
+--file=/luminesce-examples/simplified_valuation_data.xlsx
+--worksheet=instruments
+enduse;
+
+-- Run instruments transformation
+@instruments_for_upload = select
+name as DisplayName,
+figi as Figi,
+'GBP' as DomCcy
```
[the remark this answers is not in the document]
Comment (slemasne) [reply examples/lusid/run-valuation/1-create-instruments.sql:12]: It should be in file. I will add.

File: examples/lusid/run-valuation/2-create-portfolio.sql [hunk 8]
```diff
@@ -0,0 +1,14 @@
+-- Defining scope and code variables
```
[the remark this answers is not in the document]
Comment (slemasne) [reply examples/lusid/run-valuation/2-create-portfolio.sql:8]: done.

File: examples/lusid/run-valuation/4-upload-transactions.sql [hunk 9]
```diff
@@ -0,0 +1,30 @@
+-- Load transaction from Excel file on Drive
```
[the remark this answers is not in the document]
Comment (slemasne) [reply examples/lusid/run-valuation/4-upload-transactions.sql:8]: done.

File: examples/lusid/run-valuation/5-run-simple-valuation.sql (hunk 1 above)
[the remark this answers is not in the document]
Comment (slemasne) [reply examples/lusid/run-valuation/5-run-simple-valuation.sql:10]: done.

File: examples/lusid/run-valuation/2-create-portfolio.sql (hunk 3 above)
[the remark this answers is not in the document]
Comment (slemasne) [reply examples/lusid/run-valuation/2-create-portfolio.sql:15]: updated

**Review by DavidWassellFinbourne — APPROVED** [review]
Looks good

**Review by DavidWassellFinbourne — APPROVED** [review]

**Pull request merged**